install-wp-tests.sh

```diff
@@ -149,9 +149,9 @@ download() {
   fi
 
   if command -v curl >/dev/null 2>&1; then
-    curl -f -s "$1" > "$2"
+    curl -f -s -L "$1" > "$2"
```

Comment:
Is there any risk to following redirects here?

Reply:
They're all from known URLs